runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -300,17 +303,25 @@ public void invalidateClientCache() {
      * @return A result to be cached
      */
     public @Nonnull Map<Long, ILogData> cacheFetch(Iterable<Long> addresses) {
-        //turn the addresses into Set for now to satisfy signature requirement down the line
-        Set<Long> readAddresses = new TreeSet<>();
-        Iterator<Long> iterator = addresses.iterator();
-        while(iterator.hasNext()){
-            readAddresses.add(iterator.next());
+        Map<Long, ILogData> temp = new HashMap<>();
+
+        int batchSize = 20;
```

Comment:
This constant seems like something that someone soon will want to make configurable/adjustable?